### ISSUE TITLE
Can't install vmulti driver

### ISSUE THREAD

**Issue by 6Digit** [hawku/TabletDriver#379]
## Hardware/software information
Windows Version: Windows 10 64-bit

Tablet Driver version: devocub edition v0.1.5.5

Tablet model: cth 490

Whenever I press enter to install the vmulti driver, the window suddenly disappears. I have checked another post with the same issue but did not find a fix, help?

**Comment by InfinityGhost** [comment]
Provide some screenshots and logs if possible to help diagnose

**Comment by 6Digit** [comment]
> Provide some screenshots and logs if possible to help diagnose

will this do? : https://i.gyazo.com/57d96818729feaabc4d748a9edb6ee4d.gif?_ga=2.149430527.878875188.1551139790-1674263470.1551139790

**Comment by InfinityGhost** [comment]
Odd, was it running as admin?

**Comment by 6Digit** [comment]
> Odd, was it running as admin?

i'm unsure

**Comment by 6Digit** [comment]
let me try again

**Comment by InfinityGhost** [comment]
Run the vmulti driver installer as admin is what I meant

**Comment by ItsLemonTime** [comment]
Heyo. I'm also having the same issue. The install window immediately disappears. If I run it as not-administrator, the window shows some shit, and then asks me for access, and then disappears.

**Comment by 6Digit** [comment]
> Run the vmulti driver installer as admin is what I meant

is it possible to install it when it is not in the folder?

**Comment by 6Digit** [comment]
still disappears

**Comment by InfinityGhost** [comment]
Well really we've narrowed this down to an issue with the vmulti driver, and not TabletDriver

**Comment by 6Digit** [comment]
Is there anything that I can do for the time being in an attempt to fix it?

**Comment by InfinityGhost** [comment]
At the moment, I have no clue besides trying to install it in compatibility mode or something

**Comment by 6Digit** [comment]
ok
thank you for your help, I will continue to wait for a fix but I guess I will have to go back to my old drivers

**Comment by qbuss** [comment]
1. disconnect the tablet (so the windows won't install the drivers automaticly).
2. uninstall wacom drivers.
3. isntall hawku drivers as admin
4. open tabletdriver (make sure that it's unpacked in the same directory as all the other files)
so it looks like this.
![image](https://user-images.githubusercontent.com/12773343/53638918-175f7100-3c28-11e9-8d88-7d83d7d1e462.png)

**Comment by Rangga676** [comment]
I ran the install_vmulti_64.bat in the driver_vmulti folder with admin and it works

**Comment by Xxsh4d0wsxX** [comment]
i cant install it when i do it will spam CMD with this message

C:\WINDOWS\system32>REM  --> Check for permissions

C:\WINDOWS\system32>"C:\WINDOWS\system32\cacls.exe" "C:\WINDOWS\system32\config\system"   1>nul 2>&1

C:\WINDOWS\system32>REM --> If error flag set, we do not have admin.

C:\WINDOWS\system32>if '2' NEQ '0' (
echo Requesting administrative privileges...
 goto UACPrompt
)  else (goto gotAdmin  )
Requesting administrative privileges...

C:\WINDOWS\system32>echo Set UAC = CreateObject("Shell.Application")    1>"C:\Users\username~1\AppData\Local\Temp\getadmin.vbs"

C:\WINDOWS\system32>echo UAC.ShellExecute "C:\Users\username~1\DOWNLO~1\COMPRE~1\TABLET~1.3\DRIVER~3\INSTAL~1.BAT", "", "", "runas", 1    1>>"C:\Users\username~1\AppData\Local\Temp\getadmin.vbs"

someone plz help

**Comment by WhySiska** [comment]
вылетает такая ошибка:
![image](https://user-images.githubusercontent.com/79080464/107923079-09749900-6f82-11eb-99e6-5c9845a655f2.png)
